src/frontend/src/handler/create_table.rs

```diff
@@ -1349,6 +1349,29 @@ pub async fn handle_create_table(
 ) -> Result<RwPgResponse> {
     let session = handler_args.session.clone();
 
+    // CDC connector ban check
```

Comment:
I suggest we put the checking logic in [check_create_table_with_source](https://github.com/risingwavelabs/risingwave/blob/b6524f9d6ad9156d54afb47194bfd7d59353e153/src/frontend/src/handler/create_table.rs#L1026) so that `EXPLAIN` command can also work.